docs/platforms/native/configuration/transports.mdx

```diff
@@ -69,11 +69,8 @@ sentry_init(options);
 /* ... */
 ```
 
+### Proxy behavior
 
-We support `HTTP` proxies on all platforms, and `SOCKS5` proxies on Linux and macOS. Depending on the platform, the transport provides a fallback in case the proxy is not reachabled. The following table shows the expected behaviour.
+We support `HTTP` proxies on all platforms, and `SOCKS5` proxies on Linux and macOS. On Windows, `https` proxy servers are not supported by the default transport (WinHTTP). The proxy may also contain authentication (`"http://user:password@my.proxy:8080"`) or be written in IPv6 format (`"http://[::1]:8080"`).
 
-| Platform | http-proxy                                          | socks-proxy                                          |
-|----------|-----------------------------------------------------|------------------------------------------------------|
-| Windows  | Internal: fallback <br /> Crashpad: fallback        | N/A                                                  |
-| Linux    | Internal: fallback <br />  Crashpad: fallback       | Internal: no fallback <br />  Crashpad: no fallback  |
-| macOS    | Internal: no fallback <br />  Crashpad: no fallback | Internal: no fallback <br />  Crashpad: fallback     |
+Depending on the [DSN](https://docs.sentry.io/concepts/key-terms/dsn-explainer/) protocol (`https` for Sentry projects), the matching `http(s)_proxy` environment variable is read. To disable this, use the `sentry_options_set_proxy(options, "")` function with an empty proxy string argument.
```

Comment:
Maybe it's nicer if the code snippet is not inline

Reply:
> Maybe it's nicer if the code snippet is not inline

Agreed. I would also reframe the paragraph to highlight that we read the environment variable first and defaulting behavior second, something like the following:

```suggestion
The SDK will always automatically configure the HTTP proxy if a corresponding environment variable is set. Depending on the [DSN](https://docs.sentry.io/concepts/key-terms/dsn-explainer/) protocol, the matching `http(s)_proxy` environment variable is read, meaning you must set the `https_proxy` variable for a usual Sentry-hosted `DSN`.

If you do not want the SDK to configure the HTTP proxy from the environment variables automatically, use the `sentry_options_set_proxy(options, "")` function with an empty proxy string argument.
```